docs/setup/install/kubestash/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Install KubeStash
+description: Installation guide for KubeStash
+menu:
+  docs_{{ .version }}:
+    identifier: install-kubestash-enterprise
+    name: KubeStash
+    parent: installation-guide
+    weight: 20
+product_name: kubestash
+menu_name: docs_{{ .version }}
+section_menu_id: setup
+---
+
+# Install KubeStash
+
+If you are willing to try KubeStash, you can grab a **30 days trial** license from [here](https://license-issuer.appscode.com/?p=kubestash).
+
+
+### Get a Trial License
+
+
+In this section, we are going to show you how you can get a **30 days trial** license for KubeStash. You can get a license for your Kubernetes cluster by going through the following steps:
+
+
+- At first, go to [AppsCode License Server](https://license-issuer.appscode.com/?p=stash-enterprise) and fill up the form. It will ask for your Name, Email, the product you want to install, and your cluster ID (UID of the `kubestash` namespace).
+- Provide your name and email address. **You must provide your work email address**.
+- Then, select `KubeStash License` in the product field.
```

Comment:
There is no `KubeStash License` field in the license website. It is `Stash License`.